packages/api/controllers/hackathon_register_controller.go

```diff
@@ -51,10 +51,11 @@ func RegisterTeamMember(c *fiber.Ctx) error {
 		ShareInfoWithSponsors:          member.ShareInfoWithSponsors,
 		WantJobOffers:                  member.WantJobOffers}
 
-	numberOfMembers, _ := GetNumberOfHackathonParticipants(ctx)
+	numberOfParticipantsWithTeam, _ := GetNumberOfHackathonParticipants(ctx)
+	numberOfParticipantsWithoutTeam, _ := GetNumberOfNoTeamParticipants(ctx)
 	numberOfTeams, _ := GetNumberOfHackathonTeams(ctx)
 
-	if numberOfMembers >= 90 {
+	if numberOfParticipantsWithTeam+numberOfParticipantsWithoutTeam >= 90 {
```

Comment:
Why don't you just extract this in one function? Either way we would always need to traverse through the two collections. 

e.g. 
```
number_of_participants = GetTotalParticipants()
```

Reply:
ok

Reply:
After the registration and probably the hackathon I would try to do a refactoring of the whole backend for the registration.